src/requirements.txt

```diff
@@ -1,3 +1,4 @@
-azure-mgmt-netapp>=0.6.0
-azure-mgmt-resource>=3.1.0
+azure-mgmt-netapp>=3.0.0
```

Comment:
Just a small point, a newer release was just published yesterday, https://pypi.org/project/azure-mgmt-netapp/4.0.0/

Reply:
LGTM as well, might want consider a bump to latest as Hodd says

Reply:
I attempted to update to version 4.0.0, but the SDK no longer works when it tries to create a volume from a snapshot, and gives this error:

An error ocurred. Error details: (Conflict) Operation: 'Change Ownership' is not permitted.
Code: Conflict
Message: Operation: 'Change Ownership' is not permitted.
Exception Details:      (OperationNotPermitted) Operation: 'Change Ownership' is not permitted.
        Code: OperationNotPermitted
        Message: Operation: 'Change Ownership' is not permitted.
Traceback (most recent call last):
  File "C:\Users\t-cassidyf\AppData\Local\Programs\Python\Python39\lib\site-packages\azure\core\polling\base_polling.py", line 482, in run        
    self._poll()
  File "C:\Users\t-cassidyf\AppData\Local\Programs\Python\Python39\lib\site-packages\azure\core\polling\base_polling.py", line 521, in _poll
    raise OperationFailed("Operation failed or canceled")
azure.core.polling.base_polling.OperationFailed: Operation failed or canceled

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "c:\Users\t-cassidyf\sdk-samples\netappfiles-python-sdk-sample\src\example.py", line 718, in <module>
    run_example()
  File "c:\Users\t-cassidyf\sdk-samples\netappfiles-python-sdk-sample\src\example.py", line 384, in run_example
    volume_from_snapshot = create_volume_from_snapshot(anf_client,
  File "c:\Users\t-cassidyf\sdk-samples\netappfiles-python-sdk-sample\src\example.py", line 208, in create_volume_from_snapshot
    return client.volumes.begin_create_or_update(resource_group_name,
  File "C:\Users\t-cassidyf\AppData\Local\Programs\Python\Python39\lib\site-packages\azure\core\polling\_poller.py", line 255, in result
    self.wait(timeout)
  File "C:\Users\t-cassidyf\AppData\Local\Programs\Python\Python39\lib\site-packages\azure\core\tracing\decorator.py", line 83, in wrapper_use_tracer
    return func(*args, **kwargs)
  File "C:\Users\t-cassidyf\AppData\Local\Programs\Python\Python39\lib\site-packages\azure\core\polling\_poller.py", line 275, in wait
    raise self._exception # type: ignore
  File "C:\Users\t-cassidyf\AppData\Local\Programs\Python\Python39\lib\site-packages\azure\core\polling\_poller.py", line 192, in _start
    self._polling_method.run()
  File "C:\Users\t-cassidyf\AppData\Local\Programs\Python\Python39\lib\site-packages\azure\core\polling\base_polling.py", line 500, in run        
    raise HttpResponseError(
azure.core.exceptions.HttpResponseError: (Conflict) Operation: 'Change Ownership' is not permitted.
Code: Conflict
Message: Operation: 'Change Ownership' is not permitted.
Exception Details:      (OperationNotPermitted) Operation: 'Change Ownership' is not permitted.
        Code: OperationNotPermitted
        Message: Operation: 'Change Ownership' is not permitted.

It doesn't seem like anything drastic has changed from version 3.0.0 to 4.0.0, and they were only released a few weeks apart, but I can't figure out where this issue could be coming from. I looked at the changes that were listed as well as the code itself to see where it could be breaking. I also tried to debug, but the code won't let me step into the library functions either, and just gets stuck at line 384 for about a minute. It's unclear if there was an issue introduced in this library between versions, so we could just keep the version at 3.0.0, since it works at that version. Any thoughts?

Reply:
I am hitting the same issue with the Java SDK, seems like there is a bug in API 2021-04-01 with the addition of chown_mode field in ExportPolicyRule. I managed to get the Java SDK running by removing the export policy from the volume created from the snapshot.
I'll take a look at this a little better today and file a bug if my theory is correct.